nx/lib/nx.ex

```diff
@@ -1391,6 +1391,117 @@ defmodule Nx do
     eye(Nx.shape(tensor), opts)
   end
 
+  @doc """
+  Lower triangle of an array.
```

Comment:
```suggestion
  Lower triangle of a matrix.
```

And we also need to implement and document the batching behaviour. :)